src/algebra/big_operators/fin.lean

```diff
@@ -95,6 +95,30 @@ by simp
   ∏ i, f i = f 0 * f 1 :=
 by simp [prod_univ_succ]
 
+@[to_additive] theorem prod_univ_three [comm_monoid β] (f : fin 3 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 :=
+by { rw [mul_assoc], simp [prod_univ_succ] }
+
+@[to_additive] theorem prod_univ_four [comm_monoid β] (f : fin 4 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 * f 3 :=
+by { repeat {rw [mul_assoc]}, simp [prod_univ_succ], refl }
+
+@[to_additive] theorem prod_univ_five [comm_monoid β] (f : fin 5 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 * f 3 * f 4 :=
+by { repeat {rw [mul_assoc]}, simp [prod_univ_succ], refl }
+
+@[to_additive] theorem prod_univ_six [comm_monoid β] (f : fin 6 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 * f 3 * f 4 * f 5 :=
+by { repeat {rw [mul_assoc]}, simp [prod_univ_succ], refl }
+
+@[to_additive] theorem prod_univ_seven [comm_monoid β] (f : fin 7 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 * f 3 * f 4 * f 5 * f 6 :=
+by { repeat {rw [mul_assoc]}, simp [prod_univ_succ], refl }
+
+@[to_additive] theorem prod_univ_eight [comm_monoid β] (f : fin 8 → β) :
+  ∏ i, f i = f 0 * f 1 * f 2 * f 3 * f 4 * f 5 * f 6 * f 7 :=
+by { repeat {rw [mul_assoc]}, simp [prod_univ_succ], refl }
```

Comment:
Note: all these lemmas can be proved by `(multiset.fold_eq_foldl _ _ _).trans $ by { rw ← one_mul (f 0), refl }`